crowbar_framework/app/models/node_object.rb

```diff
@@ -553,10 +553,26 @@ def memory
     @node["memory"]["total"] rescue nil
   end
 
+  def architecture
+    @node["kernel"]["machine"] rescue nil
```

Comment:
Style/RescueModifier: Avoid using `rescue` in its modifier form. (https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers)